modules/mobile/docs/schemas/v1/Appointment.yml

```diff
@@ -0,0 +1,185 @@
+                example: CA
+              zipCode:
+                type: string
+                example: 90822
+          lat:
+            type: float
+            example: 33.770050
+          long:
+            type: float
+            example: -118.193741
+          phone:
+            type: object
+            required:
+              - number
+              - extension
+            properties:
+              areaCode:
+                type: string
+                example: 562
+              number:
+                type: string
+                example: 434-6008
+              extension:
+                type: string
+                example: 1001
+          url:
+            type: string
+            example: https://care2.evn.va.gov/vvc-app/?join=1&media=1&escalate=1&conference=VVC8275247@care2.evn.va.gov&pin=3242949390#
+          code:
+            type: string
+            example: GL32C
+      minutesDuration:
+        type: integer
+        example: 60
+      phoneOnly:
+        type: boolean
+        example: true
+      startDateLocal:
+        type: datetime
+        example: 2019-04-20T14:15:00.000-04:00
+      startDateUtc:
+        type: datetime
+        example: 2019-04-20T18:15:00.000Z
+      status:
+        type: string
+        description: Booking status of the appointment. Note that 'HIDDEN' means that the status should be hidden, the appointment should still be visible.
+        enum: [ BOOKED, CANCELLED, HIDDEN, SUBMITTED ]
+        example: BOOKED
+      statusDetail:
+        type: string
+        description: For a cancelled appointment return details about who or why it was cancelled.
+        enum: [ CANCELLED BY CLINIC & AUTO RE-BOOK,
+                CANCELLED BY CLINIC,
+                CANCELLED BY PATIENT & AUTO-REBOOK,
+                CANCELLED BY PATIENT,
+                CANCELLED - OTHER ]
+        example: BOOKED
+      timeZone:
+        type: string
+        example: "America/Los_Angeles"
+      vetextId:
+        type: string
+        example: 308;20210106.140000
+      reason:
+        type: string
+        example: "Follow-up/Routine: Reason 1"
+      isCovidVaccine:
+        type: boolean
+        example: false
+      isPending:
+        type: boolean
+        example: false
+      proposedTimes:
+        type: array
+        example: [{ date: "10/01/2020", time: "AM" }, { date: "10/01/2020", time: "PM" }, { date: null, time: null }]
+      typeOfCare:
+        type: string
+        example: "Primary Care"
+      patientPhoneNumber:
+        type: string
+        example: "123-456-7890"
+      patientEmail:
+        type: string
+        example: "someone@example.com"
+      bestTimeToCall:
+        type: array
+        example: ["Morning", "Afternoon", "Evening"]
+      friendlyLocationName:
+        type: string
+        example: "CHYSHR-Cheyenne VA Medical Center"
+      serviceCategoryName:
+        type: string
+        example: "COMPENSATION & PENSION"
+      appointmentIEN:
```

Comment:
It looks like we started by sorting fields alphabetically but when we added new ones, we tacked them on the end. While FE does use the key values to transverse objects, it seems safest to stick with this strategy in case they have any hidden code that looks at index of keys.